charactersheet/charactersheet/viewmodels/ability_scores.js

```diff
@@ -65,4 +67,17 @@ function AbilityScoresViewModel() {
         self.abilityScores().save();
         Notifications.abilityScores.changed.dispatch();
     };
+
+    // Modal Methods
+
+    self.openModal = function() {
+        self.modalStatus(true);
+         // Alert the modal even if the value didn't technically change.
+        self.modalStatus.valueHasMutated();
+    };
+
+    self.modalFinishedAnimating = function() {
```

Comment:
Is there any way we can make these methods available to all VMs? If not, we're going to have all this boilerplate for every VM with a modal.